clang/test/CodeGen/PR76674.cpp

```diff
@@ -0,0 +1,11 @@
+// RUN: %clang_cc1 -verify -std=c++20 -emit-llvm -o - %s
```

Comment:
I wonder if `-verify` flag is necessary? There is no diagnostics expected, and the test makes sure LLVM IR generation doesn't crash even without `-verify` flag.

Reply:
removed `-verify`